File: Robbie/wk01/3-wed/arrays.js
// 1.Create an array of the days of the week

var days_of_the_week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];

// 2. My calendar says the first day is Sunday...


var sd = days_of_the_week.pop();
days_of_the_week.unshift(sd);


// 3. Create a new array of the days of the week:

var weekdays = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'];
var weekend = ['Saturday', 'Sunday'];
days_of_the_week = [weekdays, weekend];
//var weekdays = days_of_the_week.slice(0, 5);
//var weekend = days_of_the_week.slice(5, 7);

//days_of_the_week = weekdays.concat(weekend);

// 4. Remove either the weekdays or the weekends

days_of_the_week.pop();
days_of_the_week = days_of_the_week[0];

// 5. Sort the remaining days alphabetically
console.log(days_of_the_week.sort());

File: Robbie/wk04/1-mon/ruby1/arrays.rb
require 'pry'

days_of_the_week = %w(Monday Tuesday Wednesday Thursday Friday Saturday Sunday)
puts 'Q1'
puts days_of_the_week
puts 'Q2'
puts days_of_the_week.rotate(-1)
puts 'Q3'
weekdays = %w(Monday Tuesday Wednesday Thursday Friday)
weekend = %w(Saturday Sunday)
new_days_of_the_week = [weekdays,weekend]
puts new_days_of_the_week
puts 'Q4'
taken_days = new_days_of_the_week.shift
puts taken_days
puts new_days_of_the_week
puts 'Q5'
puts new_days_of_the_week.sort

binding.pry